packages/cli/src/commands/generate/service/templates/test.ts.template

```diff
@@ -47,24 +58,24 @@ describe('${pluralCamelName}', () => {
     expect(result).toEqual(scenario.${singularCamelName}.one)
   })
 
-  <% if (create) { %>scenario('creates a ${singularCamelName}', async (${stringifyValue(create).includes('scenario.') ? 'scenario: StandardScenario' : ''}) => {
+  <% if (create) { %>scenario('creates a ${singularCamelName}', async (${extractValues(create).includes('scenario.') ? 'scenario: StandardScenario' : ''}) => {
     const result = await create${singularPascalName}({
-      input: ${stringifyValue(create)},
+      input: ${extractValues(create)},
     })
 
     <% for (const [name, value] of Object.entries(create)) { %>
-    expect(result.${name}).toEqual(${stringifyValue(value, true)})<% } %>
+    expect(result.${name}).toEqual(${extractExpectedValue(value)})<% } %>
```

Comment:
Having a thin wrapper around `extractValues` like you have here works really well! Makes the code easy to read and understand 👌